Explain openapi boolean option (#1590)
api-platform/docs: core/openapi.md

To pass a context to the OpenAPI **v2** generator, use the `swaggerContext` attribute (notice the prefix: `swagger` instead of `openapi`).

> ## Disabling an operation from OpenAPI documentation

Reviewer: ```suggestion
## Disabling an Operation From OpenAPI Documentation
```

> Sometimes you may want to disable an operation from the OpenAPI documentation, for example to don't expose it.

Reviewer: ```suggestion
Sometimes you may want to disable an operation from the OpenAPI documentation, for example to not exposing it.
```

> Using the `openapi` boolean option disable this operation from the OpenApi documentation:

Reviewer: ```suggestion
Using the `openapi` boolean option disables this operation from the OpenAPI documentation:
```

[codeSelector]

```php
<?php
// api/src/Entity/Product.php
namespace App\Entity;

use ApiPlatform\Metadata\ApiResource;
use ApiPlatform\Metadata\GetCollection;

#[ApiResource(
    operations: [
        new GetCollection(openapi: false)
    ]
)]
class Product
{
    // ...
}
```

```yaml
# api/config/api_platform/resources.yaml
resources:
    App\Entity\Product:
        operations:
            ApiPlatform\Metadata\GetCollection:
                openapi: false
```

```xml
<?xml version="1.0" encoding="UTF-8" ?>
<!-- api/config/api_platform/resources.xml -->

<resources xmlns="https://api-platform.com/schema/metadata/resources-3.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0
           https://api-platform.com/schema/metadata/resources-3.0.xsd">
    <resource class="App\Entity\Product">
        <operations>
            <operation class="ApiPlatform\Metadata\GetCollection" openapi="false" />
        </operations>
    </resource>
</resources>
```

[/codeSelector]

> Note: as your route is not exposed, you may want to return a HTTP 404 if it's called. Prefer using the NotExposedAction controller.

Reviewer: ```suggestion
Note: as your route is not exposed, you may want to return a HTTP 404 if it's called. Prefer using the `NotExposedAction` controller instead.
```
